Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PowderDiffILLDetScanReduction.py

```diff
@@ -155,16 +157,17 @@ def _reduce_2D(self, input_group):
 
         return output2D
 
-    def PyExec(self):
+    def PyExec(self): # noqa C901
```

Comment:
I don't think you have a justified reason to skip this check. You can look at breaking things out like getting the components, doing the normalisation etc.